Before: 838b66d1840135c5a59cd7a243d996dae4fe2f306a20ee082d6bdf4b05d583f4
After: dd34a4f6ca3c2ff29021dcf53aeaafba13c7b3531a5b4002e734875ac6a45027
Patch: 884ff173e2dc0cdc77de93684386bf072c39fc248ff2ab533e082a8ab7b311a1
--- Rakefile	2012-07-22 20:44:49.448916158 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,6 +0,0 @@
-raise "Rake must not be run directly. Either run via Bundler (bundle exec rake) or via bin/rake." unless defined?(Bundler)
-APP_ROOT ||= File.expand_path(File.dirname(__FILE__))
-require 'corvid/rake'
-
-# Set default task to test
-# task :default => :test
